Resources/Prototypes/Entities/Objects/Devices/translator_implants.yml

```
@@ -126,3 +126,18 @@
     - Moffic
     requires:
     - TauCetiBasic
+
+- type: entity
+  parent: BaseSubdermalImplant
+  id: ValyrianStandardTranslatorImplant
+  name: valyrian standard translator implant
+  description: An implant giving the ability to understand and speak Valyrian Standard. Chirp!
+  noSpawn: true
+  components:
+  - type: TranslatorImplant
```

yamllint

[error] wrong indentation: expected at least 3

(indentation)